packages/shared/src/components/checklist/ChecklistCard.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { ReactElement, useMemo, useState } from 'react';
+import classNames from 'classnames';
+import { Card } from '../cards/Card';
+import MiniCloseIcon from '../icons/MiniClose';
+import { IconSize } from '../Icon';
+import { ChecklistStep } from './ChecklistStep';
+import { ChecklistAction, ChecklistCardProps } from '../../lib/checklist';
+
+const ChecklistCard = ({
+  className,
+  title,
+  description,
+  steps,
+  onRequestClose,
+}: ChecklistCardProps): ReactElement => {
+  const activeStep = useMemo(
+    () => steps.find((item) => !item.action.dateCompleted)?.action.type,
+    [steps],
+  );
+  const [checkedStep, setCheckedStep] = useState<string>(activeStep);
+  const isDone = useMemo(
+    () => steps.every((item) => !!item.action.dateCompleted),
+    [steps],
+  );
+
+  const onToggleStep = (action: ChecklistAction) => {
+    setCheckedStep((currentCheckedStep) => {
+      if (currentCheckedStep === action.type) {
+        return undefined;
+      }
+
+      return action.type;
+    });
+  };
+
+  return (
+    <div className={className}>
+      <Card className="p-0 rounded-14 border-cabbage-40 hover:!border-cabbage-40 w-[340px]">
+        <div className="p-4 rounded-t-14 bg-theme-bg-cabbage-opacity-24">
+          <p className="mb-1 font-bold typo-body text-theme-label-primary">
+            {title}
+          </p>
+          <p className="text-salt-90 typo-callout">{description}</p>
+          {typeof onRequestClose === 'function' && (
+            <button
+              className="absolute top-4 right-4 text-salt-90"
```

Comment:
Same note

Reply:
@rebelchris so for this the right value should be `text-theme-color-salt`?